nemesis/arena_prep.py

```diff
@@ -233,13 +146,13 @@ def return_to_airship():
         memory.main.wait_frames(1)
     xbox.menu_b()
     memory.main.await_control()
-    print("Return to Airship Complete.")
+    logger.debug("Return to Airship Complete.")
     memory.main.clear_save_menu_cursor()
     memory.main.clear_save_menu_cursor_2()
 
 
 def battle_farm_all(ap_cp_limit: int = 255, yuna_attack=True, fayth_cave=True):
-    print("### Battle Start:", memory.main.get_encounter_id())
+    logger.debug(f"Battle Start:{memory.main.get_encounter_id()}")
```

Comment:
Beware that `print("something", x)` will inject a space between the string and the variable. You have to put that in manually in f-strings (if you want it to look the same).

Reply:
Thanks, I'll keep that in mind going forward. For now, planning to push forward with the PR and will circle back as needed.